docs/cloud/rahti2/images/Using_Rahti_2_integrated_registry.md

```diff
@@ -39,3 +41,41 @@ Go to the Images section, make sure the option "Deploy images from an image stre
 Finally select the new image.
 
 ![Use cached image](../../img/use_cached_image.png)
+
+## Access Control for the Rahti 2 Integrated Registry
+
+Rahti 2 allows fine-grained control over access to the integrated image registry, enabling management of access based on [user authentication](https://docs.openshift.com/container-platform/4.15/authentication/understanding-authentication.html#rbac-groups_understanding-authentication).
```

Comment:
Could the bookmark part of the url be removed? Now it takes to Group subheader but I guess it could point directly to the top header instead.